stacks-signer/src/signer.rs

```diff
@@ -928,11 +928,7 @@ impl Signer {
     ) -> std::collections::HashMap<StacksAddress, u64> {
         let mut account_nonces = std::collections::HashMap::with_capacity(signer_addresses.len());
         for address in signer_addresses {
-            let Ok(account_nonce) = retry_with_exponential_backoff(|| {
```

Comment:
I think you need to add a retry around get_account_nonces in get_expected_transactions and again in process_dkg. Then I will approve :D

Reply:
Since the inner most stacks client call was done with retry, I didn't add the outer layer retry but added "with_retry" to those client function names so it's more clear.